hudi-common/src/main/java/org/apache/hudi/parquet/io/OutputStreamBackedOutputFile.java

```diff
@@ -33,9 +34,9 @@ public class OutputStreamBackedOutputFile implements OutputFile {
 
   private static final long DEFAULT_BLOCK_SIZE = 1024L * 1024L;
 
-  private final FSDataOutputStream outputStream;
+  private final SeekableDataOutputStream outputStream;
```

Comment:
Looks like this class is using Parquet library, which can be moved to `hudi-hadoop-common` module.